src/Liquidator.sol

```diff
@@ -181,13 +190,10 @@ contract Liquidator is Owned, ILiquidator {
     /**
      * @notice Initiate the liquidation of an Account.
      * @param account The contract address of the Account to be liquidated.
-     * @dev We do not check if the address passed is an actual Arcadia Account.
-     * A malicious msg.sender can pass a self created contract as Account (not an actual Arcadia-Account),
-     * that implemented startLiquidation().
-     * This would successfully start an auction and the malicious non-Account might be in auction indefinitely,
-     * but this does not block or impact any current or future 'real' auctions of Arcadia-Accounts.
```

Comment:
i think we should add altered version of this part of natspec, can be in other PRs tho